tests/util/test_atoms.py

```diff
@@ -141,3 +142,16 @@ def test_get_highest_block():
     assert get_highest_block(atoms) == "f"
     atoms = molecule("H2O")
     assert get_highest_block(atoms) == "p"
+
+
+def test_true_charge_and_spin():
+    atoms = molecule("CH3")
```

Comment:
ASE already initialized magnetic moments on the CH3, so the correct multiplicity for `molecule("CH3")` is likely set correctly even without your function.

Could you add one more test:

```
atoms = molecule("CH3")
atoms.set_initial_magnetic_moments([0.0]*len(atoms))
```

Copying and pasting your existing test with this additional `atoms` object.